app/src/main/java/com/sdp13epfl2021/projmag/MainActivity.kt

```diff
@@ -22,7 +22,6 @@ class MainActivity : AppCompatActivity() {
         //These are used as name for extra in Intent
         const val fromLinkString: String = "fromLink"   //Boolean
         const val projectIdString: String = "projectId" //ProjectId
-        const val projectString: String = "project"     //ImmutableProject
```

Comment:
Maybe better to keep it here.

Reply:
I removed it because it was redundant but ok